onnx/test/shape_inference_test.py

```diff
@@ -8491,291 +8492,88 @@ def test_blackmanwindow(self):  # type: () -> None
-                            3,
-                            3,
-                            3,
-                            3,
-                            3,
-                            4,
-                            4,
-                            4,
-                            4,
-                            4,
-                        ),
-                    ),
-                ),
-                make_node("DFT", ["input", ""], ["output"], axis=1, onesided=1),
-            ],
-            [],
-        )
-        self._assert_inferred(
-            graph,
-            [
-                make_tensor_value_info("shape", TensorProto.FLOAT, (1, 5, 10, 1)),
-                make_tensor_value_info("y", TensorProto.FLOAT, (1, 3, 10, 2)),
-            ],
-        )  # type: ignore
-
-        graph = self._make_graph(
-            [],
-            [
-                make_node(
-                    "Constant",
-                    [],
-                    ["input"],
-                    value=make_tensor(
-                        "input",
-                        TensorProto.FLOAT,
-                        (
-                            1,
-                            5,
-                            10,
-                            1,
-                        ),
-                        (
-                            0,
-                            0,
-                            0,
-                            0,
-                            0,
-                            1,
-                            1,
-                            1,
-                            1,
-                            1,
-                            2,
-                            2,
-                            2,
-                            2,
-                            2,
-                            3,
-                            3,
-                            3,
-                            3,
-                            3,
-                            4,
-                            4,
-                            4,
-                            4,
-                            4,
-                            0,
-                            0,
-                            0,
-                            0,
-                            0,
-                            1,
-                            1,
-                            1,
-                            1,
-                            1,
-                            2,
-                            2,
-                            2,
-                            2,
-                            2,
-                            3,
-                            3,
-                            3,
-                            3,
-                            3,
-                            4,
-                            4,
-                            4,
-                            4,
-                            4,
-                        ),
-                    ),
+    @parameterized.expand(
```

Comment:
Should add another test-case where the axis-value is unknown (a graph-input).

Reply:
Done